app/components/base/TransferWorkflow.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import isEqual from "lodash.isequal";
+import FundsFlowSelector from "./FundsFlowSelector";
+import Button from "./Button";
+import { GREY_1, GREY_3 } from "../colors";
+import { breakDown } from "../breakpoints";
+import VCRBanktoVCRBankdata from "../../../assets/transfer-workflow-data/vcr-bank-to-vcr-bank.json";
+
+// styles
+
+const StyledContainer = styled.div`
+  max-width: 791px;
+  border: 1px solid ${GREY_3};
+  box-sizing: border-box;
+  border-radius: 5px;
+  margin: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  @media (${breakDown("md")}) {
+    width: auto;
+  }
+  @media (${breakDown("sm")}) {
+    width: 100%;
+    margin: 0;
+  }
+`;
+
+const StyledTimelineContainer = styled.div`
+  width: 100%;
+  height: 81px;
+  background: ${GREY_1};
+  border-bottom: 1px solid ${GREY_3};
+  box-sizing: border-box;
+  border-radius: 4px 4px 0px 0px;
+  margin-bottom: 50px;
+`;
+
+const StyledBottomContainer = styled.div`
+  width: 100%;
+  height: 81px;
+  border-top: 1px solid ${GREY_3};
+  box-sizing: border-box;
+  border-radius: 0px 0px 4px 4px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 12px;
+  padding: 20px;
+  margin-top: 50px;
+`;
+
+// selectable options for FundsFlowSelector component
+
+const senderTypeOptions = [
+  // { value: "account", label: "Account" },
+  { value: "vcr", label: "Verified Customer" },
+  // { value: "cr", label: "Unverified Customer" },
+];
+
+const senderSourceOptions = [
+  // { value: "balance", label: "Balance" },
+  { value: "bank", label: "Bank" },
+  // { value: "r01", label: "R01-Bank" },
+];
+
+const receiverTypeOptions = [
+  // { value: "account", label: "Account" },
+  { value: "vcr", label: "Verified Customer" },
+  // { value: "cr", label: "Unverified Customer" },
+  // { value: "ro", label: "Receive-Only User" },
+];
+
+const receiverDestinationOptions = [
+  // { value: "balance", label: "Balance" },
+  { value: "bank", label: "Bank" },
+  // { value: "r03", label: "R03-Bank" },
+  // { value: "card", label: "Debit Card" },
+];
+
+// possible Funds Flow combinations
+
+const fundsFlowCombinations = {
+  one: ["vcr", "bank", "vcr", "bank"],
+  // all possible comibnations as arrays, for example,
+  // two: ["vcr", "balance", "vcr", "bank"]
+  // three: ["cr", "bank", "vcr", "bank"]
```

Comment:
Will need to fill out all possible combinations while working on gathering data later on.